build_exercise_proj/CMakeLists.txt

```
@@ -1,24 +1,25 @@
-cmake_minimum_required(VERSION 3.1)
-project(build_exercise)
+cmake_minimum_required(VERSION 3.1) 
+project(build_exercise) 
```

It is a good idea to use the extended API.
This will communicate with both CMake generator and future readers of the code what is the expected language that you are targetting.

The "target version" is optional, but it is a nice addition.

```suggestion
project(
    build_exercise 
    LANGUAGES CXX
    VERSION 1.0.0
)
```